schedule/yast/addon-module-http@ppc64le-hmc-single-disk.yaml

```diff
@@ -0,0 +1,27 @@
+---
+name: addon-module-http@ppc64le-hmc-single-disk
```

Comment:
Do we actually need separate schedule here as you have introduced conditional schedule? We have `reconnect_mgmt_console` module on top, but `grub_test` is also there, so there is no conflict.

Reply:
i was thinking the same. both of them (regular and hmc-single-disk) could be merged into one. I just though that it would be simpler to make each one separate for each job. But i could do one for hmc-single-disk and one for the rest. with this said what do you say about the name? are `addon-module@ppc64le-hmc-single-disk.yaml` and `addon-module.yaml` ok for them?

Reply:
In general I'm against conditional schedule, so as you use it already, why not to use it for all the scenarios? And it will be trivial by just adding another conditional schedule for `BACKEND=pvm_hmc`. That would be consistent. As an alternative we could have separate schedules for each special case. I don't see a good reason for mixing both approaches here. WDYT?

For naming, I've mainly used just `_pvm` postfix. This machine name is long and ugly.

Reply:
it is one thing that i dont undertand. Do you prefer one yaml for all or two (one for _http and one for _ftp)?

Reply:
Definitely not one for ftp and http, so at least 2. So what I am proposing that you create 2 schedules instead of 4. Or we create 6 in total to handle s390x and powerVM in separate schedules. What is important for me is consistency. And in case you insist on this solution, I'm also would be fine with that.

Reply:
no, i am already have changed them in two files, if this is ok

Reply:
Done. jobgroup updated accordantly